File: pset1/mario-less/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    // Get height from user between 1-8
    int height;
    do
    {
        height = get_int("Enter height between 1-8: ");
    }
    while (height < 1 || height > 8);


    //Height of pyramid
    for (int a = 0; a < height; a++)
    {

        //Spaces
        for (int b = height - 1; b > a; b--)
        {
            printf(" ");
        }

        //Hashes
        for (int c = 0; c <= a; c++)
        {
            printf("#");
        }

        printf("\n");
    }
}

File: pset6/sentimental-mario-less/mario.py
from cs50 import get_int


# Get input from the user between 1-8
while True:
    height = get_int("Height: ")
    if height >= 1 and height <= 8:
        break

# Height of pyramid
for i in range(height):

    # Spaces
    for j in range(height - i - 1):
        print(" ", end="")

    # Hashes
    for k in range(i + 1):
        print("#", end="")
    print()